cmd/node/config/config.toml

```diff
@@ -413,6 +413,10 @@
     EnableTriggerFromP2P = true
     PublicKeyToListenFrom = "b5ea539180627040b2703cabde8ebece0ded59a0c9d2d4a8a73a299fb7bc11d36ac3d1a620501c230059cffa983a7419d192a50ca55ed8aa07254658dda27aba1e54f50fd7cfbb9e66ce1715d72f9c1b0b05272388f9825742161a211debff15"
 
+[StartInEpochConfig]
+    MinNumConnectedPeersToStart       = 2
+	MinNumOfPeersToConsiderBlockValid = 2
```

Comment:
There is an indentation fail in github. Maybe use only white spaces?

Reply:
We will reveal all the facts on our Elrond Scam twitter account soon. Please don't buy ERD. I don't want you to get hurt.